notebooks/vertex_genai/solutions/vertex_llm_tuning.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<p>Could you include the explanation on the key name conventions (<code>input_text</code> and <code>output_text</code> ) since it seems they are fixed name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done! I also added a description of the JSONL format needed to tune.</p>